tdrs-backend/tdpservice/parsers/schema_defs/tanf/t5.py

```diff
@@ -0,0 +1,81 @@
+                    required=True, validators=[]),
+              Field(item="6", name='CASE_NUMBER', type='string', startIndex=8, endIndex=19,
+                    required=True, validators=[]),
+              Field(item="14", name='FAMILY_AFFILIATION', type='number', startIndex=19, endIndex=20,
+                    required=True, validators=[]),
+              Field(item="15", name='DATE_OF_BIRTH', type='number', startIndex=20, endIndex=28,
+                    required=True, validators=[]),
+              EncryptedField(decryption_func=tanf_ssn_decryption_func, item="16", name='SSN', type='string',
+                             startIndex=28, endIndex=37, required=True, validators=[]),
+              Field(item="17A", name='RACE_HISPANIC', type='number', startIndex=37, endIndex=38,
+                    required=True, validators=[]),
+              Field(item="17B", name='RACE_AMER_INDIAN', type='number', startIndex=38, endIndex=39,
+                    required=True, validators=[]),
+              Field(item="17C", name='RACE_ASIAN', type='number', startIndex=39, endIndex=40,
+                    required=True, validators=[]),
+              Field(item="17D", name='RACE_BLACK', type='number', startIndex=40, endIndex=41,
+                    required=True, validators=[]),
+              Field(item="17E", name='RACE_HAWAIIAN', type='number', startIndex=41, endIndex=42,
+                    required=True, validators=[]),
+              Field(item="17F", name='RACE_WHITE', type='number', startIndex=42, endIndex=43,
+                    required=True, validators=[]),
+              Field(item="18", name='GENDER', type='number', startIndex=43, endIndex=44,
+                    required=True, validators=[]),
+              Field(item="19A", name='REC_OASDI_INSURANCE', type='number', startIndex=44, endIndex=45,
+                    required=True, validators=[]),
+              Field(item="19B", name='REC_FEDERAL_DISABILITY', type='number', startIndex=45, endIndex=46,
+                    required=True, validators=[]),
+              Field(item="19C", name='REC_AID_TOTALLY_DISABLED', type='number', startIndex=46, endIndex=47,
+                    required=True, validators=[]),
+              Field(item="19D", name='REC_AID_AGED_BLIND', type='number', startIndex=47, endIndex=48,
+                    required=True, validators=[]),
+              Field(item="19E", name='RECEIVE_SSI', type='number', startIndex=48, endIndex=49,
+                    required=True, validators=[]),
+              Field(item="20", name='MARITAL_STATUS', type='number', startIndex=49, endIndex=50,
+                    required=True, validators=[]),
+              Field(item="21", name='RELATIONSHIP_HOH', type='string', startIndex=50, endIndex=52,
+                    required=True, validators=[]),
+              Field(item="22", name='PARENT_WITH_MINOR_CHILD', type='number', startIndex=52, endIndex=53,
```

Comment:
@elipe17 not too sure whats happening with item 22 and item 23? code looks correct:

![1109p3](https://github.com/raft-tech/TANF-app/assets/63075587/41f37848-3f6c-4af1-89a1-579079476929)

Reply:
THere was a name mismatch between the schema and the model which was causing those to be declared None even though they werent. Nice catch! I pushed the changes, once the tests run I can redeploy.

Reply:
@ADPennington, deployed to qasp and verified fields are populated. Please proceed with testing at your convenience.